### ISSUE TITLE
[CI/CD]: Must adjust the Cypress axeCheck configuration object to test for all violation levels

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13647]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The Cypress-axe configuration object is scanning for critical level violations, but is not scanning for serious, minor, or trivial violations. The configuration object must be updated to account for these other violation levels within WCAG 2 Level A and WCAG 2 Level AA rulesets to ensure the same coverage as our previous Nightwatch tests.

The axe-core configuration documents for the [options parameter](https://www.deque.com/axe/core-documentation/api-documentation/#options-parameter) says to pass in a `runOnly` object type and values. Cypress-axe docs list this as a valid way to set configuration:

> The keys consist of [those accepted by axe.run's options argument](https://www.deque.com/axe/documentation/api-documentation/#parameters-axerun) as well as a custom includedImpacts key.

```javascript
/*
 * src/platform/testing/e2e/cypress/support/commands/axeCheck.js#34
 */
const options = {
  runOnly: {
    type: 'tag',
    values: ['wcag2a', 'wcag2aa']
  }
};
```

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Cypress form tests run normally
- [ ] Tests break on any violation impact (critical, serious, minor, trivial)
- [ ] Tests do NOT break on [best-practice](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#best-practices-rules) rules
- [ ] Tests do NOT break on [experimental](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#experimental-rules) rules

## Definition of done

1. New configuration object is added
1. Tests are deployed to production
1. Main branch builds are not breaking on `axeCheck` violations

**Comment by bkjohnson** [comment]
The changes have been applied, but there were 3 apps which had failing cypress tests as a result, and they have been given a temporary exception:

- `facility-locator`
- `hca`
- `disability-benefits`

I will be making tickets and branches for each of these apps.
